File: SignatureRequests/SignatureRequests.Webs/Scripts/react/src/Entities/SignedBoxRequest.ts

class SignedBoxRequest {
  id: number;
  signedStatus: string;
  signatureId: number;
  isModel: boolean;
  text?: string;
  date?: Date;
  filePath:string;


  constructor(data: any) {
    this.id = data.id;
    this.signedStatus = data.signedStatus;
    this.signatureId = data.signatureId;
    this.isModel = data.isModel;
    this.text = data.text;
    this.date = data.date;
    this.filePath = data.filePath;
  }
}

export default SignedBoxRequest;

File: SignatureRequests/SignatureRequests.Core/RequestObjects/SignedBoxRequest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace SignatureRequests.Core.RequestObjects
{
    public class SignedBoxRequest
    {
        public int Id { get; set; }
        public string SignedStatus { get; set; }
        public int? SignatureId { get; set; }
        public bool IsModel { get; set; }
        public string Text { get; set; }
        public DateTime? Date { get; set; }

    }
}
